ember-flight-icons/CONTRIBUTING.md

```diff
@@ -54,6 +54,13 @@ In the second step you publish the package on the npm registry:
 
 _**IMPORTANT**: if you need to do some tests, use a **local** package registry (see [CONTRIBUTING](../flight-icons/CONTRIBUTING.md) in the `flight-icon`), don't test directly in production!_
 
+### How to do some manual QA of the new package version
```

Comment:
Question: Are you sure we need this? We already have two ways to test that the icons render: 

1. we have a test that explicitly checks it
2. you could run the dummy app if you wanted to see it work manually

We have tests that make sure the icons render- also running the dummy app would be a manual way to test that the icons render.

Reply:
Happy to remove. Was more adding this as a placeholder, could be `cloud-ui` in the future.

These changes https://github.com/hashicorp/flight/pull/245 worked in the dummy app but not `cloud-ui`, maybe cause of the nested `packages/`, not sure yet.

Reply:
cloud-ui PR: https://github.com/hashicorp/cloud-ui/pull/1322 but did a `yarn link` to GitHub branch

Reply:
I might be mis-understanding the purpose, let me read through it again and we can sync tomorrow 👍

Reply:
I think we can leave it for now, is a way to QA the icons "in production" (I understand we have a dummy app, but that's a more "controlled" environment, while we're not sure what happens in the real apps).
Later, when the process of releasing icons is consolidated, and we feel more confident that we're not breaking anything, we can definitely remove it.

@amyrlam I would suggest to add a paragraph/intro at the beginning of the two sections in which we explain what we're trying to do (if I am reading the CONTRIBUTING doc, I have no idea why I jump to a completely different project/repository and what that repo is for. (and maybe add is a temporary solution)

Reply:
@MelSumner @didoo i've updated the instructions and changed to `cloud-ui`. i'll merge in when CI is green. melanie can definitely make further edits when she completes https://github.com/hashicorp/flight/issues/240, it's a living document